astrokat/test/test_nd/nd-pattern-sim.yaml

```diff
@@ -1,11 +1,16 @@
 #Setting up noise diode pattern before observation
-# set noise diode pattern to fire every 20 seconds and only be on for 1.8s
+#all antennas selected, verify all responses were received
 noise_diode:
   # set pattern with all antennas
   antennas: all
-  cycle_len: 20.
-  on_frac: 0.09
+  # set noise diode pattern to fire every 16 seconds and be on for 8s
+  cycle_len: 16
+  on_frac: 0.5
```

Comment:
Correct here :)

Reply:
Not sure, but the test setups run through engineering setups variations
Perhaps you are confusing the aim of integration tests with commissioning tests